Makefile

```
@@ -52,18 +52,63 @@ build-go:
 	mkdir -p ./packages/kos-go/kos_mobile/lib/darwin-aarch64/ && \
 	cp target/release/libkos_mobile.dylib ./packages/kos-go/kos_mobile/lib/darwin-aarch64/
 
+build-go-mac-intel:
+	rustup target add x86_64-apple-darwin
+	cargo install uniffi-bindgen-go --git https://github.com/NordSecurity/uniffi-bindgen-go --tag v0.4.0+v0.28.3 && \
+	cargo build --release --target x86_64-apple-darwin --package kos-mobile && \
+	uniffi-bindgen-go --library target/x86_64-apple-darwin/release/libkos_mobile.a --out-dir ./packages/kos-go
+	mkdir -p ./packages/kos-go/kos_mobile/lib/darwin-amd64/ && \
+	cp target/x86_64-apple-darwin/release/libkos_mobile.dylib ./packages/kos-go/kos_mobile/lib/darwin-amd64/
+
 build-go-musl:
```

checkmake (0.3.2)

[warning] Target body for "build-go-musl" exceeds allowed length of 5 lines (6).

(maxbodylength)